observability/network_observability/configuring-operator.adoc

```diff
@@ -6,28 +6,29 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update the Flow Collector API resource to configure the Network Observability Operator and its managed components. The  Flow Collector is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`.
+You can update the `FlowCollector` API resource to configure the Network Observability Operator and its managed components. The  `FlowCollector` is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`. For more information, see the xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API reference].
 
 
 include::modules/network-observability-flowcollector-view.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-For more information about conversation tracking, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-working-with-conversations_nw-observe-network-traffic[Working with conversations].
+* xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API reference]
+* xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-working-with-conversations_nw-observe-network-traffic[Working with conversations]
 
 include::modules/network-observability-flowcollector-kafka-config.adoc[leveloffset=+1]
 include::modules/network-observability-enriched-flows.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-For more information about specifying flow format, see xref:../../observability/network_observability/json-flows-format-reference.adoc#network-observability-flows-format_json_reference[Network flows format reference].
+* xref:../../observability/network_observability/json-flows-format-reference.adoc#network-observability-flows-format_json_reference[Network flows format reference].
```

Comment:
Same comment:
```suggestion
* xref:../../observability/network_observability/json-flows-format-reference.adoc#network-observability-flows-format_json_reference[Network Flows format reference].
```

Reply:
I admittedly don't check this reference too often, but now that you bring my attn to this, I see there's a capitalization mismatch here too 🤦‍♀️ . I will update this: 
![image](https://github.com/openshift/openshift-docs/assets/77501157/d267a16a-045b-4475-8bf1-b92da784bb1e)